django/contrib/postgres/locale/mr/LC_MESSAGES/django.po

```diff
@@ -7,7 +7,7 @@ msgstr ""
 "Project-Id-Version: django\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2023-01-17 02:13-0600\n"
-"PO-Revision-Date: 2024-01-19 09:22+0000\n"
```

Comment:
Changes in `mr` should be reverted. Only `PO-Revision-Date` was changed in these files.

Reply:
Thank you! This was reverted and new translations were fetched.